kokoro/config/test/third_party_apps/release/template.gcl.tmpl

```diff
@@ -5,5 +5,6 @@ config build = common.third_party_apps_test {
     platforms = [{{range .ThirdPartyTestPlatforms}}
       '{{.}}',{{end}}
     ]
+    arch = '{{.Arch}}'
```

Comment:
```
{{with .Arch}}
arch = '{{.}}'
{{end}}
```
?

(So that we can continue to not specify arch for the default?)

Reply:
Fancy... OK